Cargo.toml

```diff
@@ -54,3 +54,6 @@ members = [
 default = ["cranelift-codegen/all-arch"]
 lightbeam = ["wasmtime-environ/lightbeam", "wasmtime-jit/lightbeam"]
 wasi-c = ["wasmtime-wasi-c"]
+
+# [patch.crates-io]
+# cranelift-wasm = { path = "../cranelift/cranelift-wasm" }
```

Comment:
Can you remove these lines? Or if you think it's useful to include them, please add a comment explaining what this is doing.

Reply:
I removed them. They come from https://github.com/CraneStation/wasmtime/pull/362 and I don't know any reason why they are here.

Reply:
Just something that accidentally got staged >.<